js/listeners.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+var $ = require('jquery');
+var _ = require('underscore');
+
+function Listeners() {
+  this.listeners = [];
+}
+
+Listeners.prototype.on = function(elm) {
+  var $elm = $(elm);
+  var args = _.toArray(arguments).slice(1);
+  this.listeners = this._listeners || [];
+  this.listeners.push({$elm: $elm, args: args});
+  $elm.on.apply($elm, args);
+};
+
+Listeners.prototype.off = function() {
```

Comment:
Is the intention here to always just shut off all listeners and not provide a way to shut off an individual listener, like how you can add a new one?